report_pre_thesis/src/sections/background.tex

```diff
@@ -6,7 +6,7 @@ \section{Background}\label{sec:background}
 The NASA ChemCam (Chemistry and Camera) is an advanced instrument developed by NASA in collaboration with the French national space agency (CNES).
 It is a remote-sensing laser instrument designed to analyze the chemical composition of rocks and soil on Mars\cite{chemcamNasaWebsite}.
 The instrument is mounted on the Curiosity rover and is used to collect data in the Gale Crater on Mars\cite{curiosityNasaWebsite}.
-It leverages the ChemCam-LIBS model, which uses machine learning models to analyze the spectral data obtained from ChemCam.\cite{andersonImprovedAccuracyQuantitative2017, cleggRecalibrationMarsScience2017}
```

Comment:
At this point we've already introduced the MOC model, so prefer calling it that.